test/unit/AGENTS.md

```diff
@@ -2,27 +2,27 @@
 
 ### Rules & Expectations
 
-  * Do not edit application/source files unless the refactor demands it.  Confirm before editing files outside of /test/unit, and justify why you need to make those changes.
-  * Use Sinon, not TypeMoq.  If easily possible, replace TypeMoq mocks/stubs/helpers with Sinon equivalents.
-  * Use a Sinon sandbox (setup/teardown with sinon.createSandbox()); keep helper closures (e.g., createServer) inside setup where the
-  sandbox is created.
-  * Default to chai.expect; when checking Sinon interactions, use sinon-chai.
-  * Avoid Object.defineProperty hacks and (if possible) fake/partial plain objects; use sandbox.createStubInstance(type) and sandbox.stub(obj, 'prop').value(...).
-  * Add shared Sinon helpers to test/unit/utils.ts when they’ll be reused.
-  * If updating preexisting tests, preserve relevant inline comments from the original tests.
-  * When introducing a Sinon helper to replace a TypeMoq helper (e.g., capabilities mock), follow the utils.ts pattern: accept an optional
-  sandbox, create stub instances, and return them.
-  * Avoid unnecessary casts, like `myVar as unknown as MyType` when myVar is already a sinon-stubbed instance of MyType.
-  * Maintain existing formatting conventions, line endings, and text encoding.
-  * Nest test suits as necessary to group tests in a logical manner.
+-   Do not edit application/source files unless the refactor demands it. Confirm before editing files outside of /test/unit, and justify why you need to make those changes.
```

Comment:
all .md formatting is a side-effect of the new prettier rules.  https://github.com/microsoft/vscode-mssql/pull/20275 addresses those separately, and will be pulled into this PR once merged.